presto-docs/src/main/sphinx/connector/file-based-metastore.rst

```diff
@@ -0,0 +1,78 @@
+====================
+File-Based Metastore
+====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+^^^^^^^^
+
+For testing or developing purposes, Presto can be configured to use a filesystem 
+directory as a Hive Metastore. This can be a directory on the local filesystem 
+or a non-local file system such as Amazon S3. 
+
+The file-based metastore works only with the following connectors: 
+
+* :doc:`/connector/deltalake`
+* :doc:`/connector/hive`
+* :doc:`/connector/hudi`
+* :doc:`/connector/iceberg`
+
+Configuring a File-Based Metastore
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. In ``etc/catalog/``, find the catalog properties file for the supported 
+   connector. 
+
+2. In the catalog properties file, set the following properties:
+
+.. code-block:: none
+
+    hive.metastore=file
+    hive.metastore.catalog.dir=file:///<catalog-dir>
+
+Replace ``<catalog-dir>`` in the example with the path to a directory on an 
+accessible filesystem.
+
+Using a File-Based Warehouse
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a schema
+
+.. code-block:: none
+
+    CREATE SCHEMA hive.warehouse;
+
+This query creates a folder as ``/data/hive_data/warehouse``.
```

Comment:
Let's specify that this happens if the user gave `/data/hive_data/` as the path for `hive.metastore.catalog.dir`